### PR TITLE
Count volunteer's assigned tasks

### PR DESCRIPTION
**Pull request by bronzehedwick**
This implementation gets the total amount of volunteers and sets that to
a `Map`, with the user id as the key and the count of their tasks as the
value.

This Map is then passed through the function chain to end up in dispatch
where each volunteer's id is retrieved from the Map to get the amount of
tasks assigned. This scope passing as function argument could be made
more elegant.

Also, re-work the Airtable queries to grab only the data we need,
instead of doing the filtering after the fact.

On a small note, replace the `volunteerLine` constant with a direct
`return`. While the intermediate constant is not necessary, the real
reason to do this was to make ESLint happy with max line length.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -189,10 +210,13 @@ async function checkForNewSubmissions() {
...
         const volunteers = await findVolunteers(record);
 
+        // Get the amount of tasks assigned to each volunteer
+        const volunteerTaskCounts = await getVolunteerTaskCounts();
+
         // Send the message to Slack
         let messageSent = false;
         try {
-          await sendMessage(record, volunteers);
+          await sendMessage(record, volunteers, volunteerTaskCounts);
```
Comment (ashryanbeats) [inline src/index.js:219]: In #42, `sendMessage` changes to `sendDispatch`

File: src/slack/dispatch.js [hunk 2]
```diff
@@ -163,7 +169,7 @@ const getCopyPasteNumbers = (volunteers) => {
 };
 
 // This function actually sends the message to the slack channel
-const sendMessage = async (record, volunteers) => {
+const sendMessage = async (record, volunteers, taskCounts) => {
```
Comment (ashryanbeats) [inline src/slack/dispatch.js:172]: In #42, `sendMessage` will change to `sendDispatch`

File: src/slack/dispatch.js [hunk 3]
```diff
@@ -118,7 +123,7 @@ const getAnythingElse = (record) => {
   return anythingElseObject;
 };
 
-const getVolunteers = (volunteers) => {
+const getVolunteers = (volunteers, taskCounts) => {
```
Comment (ashryanbeats) [inline src/slack/dispatch.js:126]: In #42, `getVolunteers` and all of there message formatting helper have been moved to `src/slack/message/index.js`.

File: src/slack/dispatch.js [hunk 4]
```diff
@@ -5,6 +5,11 @@ const { getDisplayNumber } = require("./phone-number-utils");
 
 const channel = config.SLACK_CHANNEL_ID;
 
+const pluralize = (num, str) => {
```
Comment (ashryanbeats) [inline src/slack/dispatch.js:8]: In #42, all message formatting helpers have been moved to `src/slack/message/index.js`. This one should likely move there as well.

**Comment by bronzehedwick** [comment]
> Thanks @bronzehedwick!
> 
> I've done a refactor of the `src/slack/` dir in #42 that will impact these changes superficially (based on a quick glance, looks like just file paths for requires and method names will change a bit here and there).
> 
> Just an FYI for now. I'm happy to pitch in on any changes to make the merge successful.

Thanks for reviewing @ashryanbeats! I'll take a swing at refactoring to your detailed explanations (thanks!). I might reach out if I get confused :)

**Pull request merged**